astro.config.mjs

```diff
@@ -10,7 +10,7 @@ export default {
     devOptions: {
         hostname: 'localhost',
         port: 3000,
-        tailwindConfig: './tailwind.config.csj',
+        tailwindConfig: './tailwind.config.cjs',
```

Comment:
good catch. Was this causing failures anywehre?

Reply:
no, it wasn't reading the config and defaulting everything AFAIK